### PR TITLE
Add tool to allow easily doing general API requests

### PR DESCRIPTION
**Pull request by RobinSikkens**
Abstracts the actual aiohttp call away from command functions by providing a tool.

Example:
```python
from devbot.tools import api_requests as API

async def extract_json():
    json = await API.get_json('http://google.com', ('name', 'henk'), ('id', 215))
```
this returns the json found on http://google.com?name=henk&id=215

The returned Jason object can be accessed like a dict

### CODE REVIEW

**Comment by romanpeters** [comment]
The example given in the PR returns a runtime error:
`sys:1: RuntimeWarning: coroutine 'get_json' was never awaited`

**Comment by RobinSikkens** [comment]
I corrected my example, thank you for bringing it up.

File: devbot/tools/api_requests.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""
+    A tool to allow for the easy aquisition of data from RESTfull API's.
+"""
+
+import aiohttp
+
+
+async def get_json(url, *params):
```
Comment (romanpeters) [inline devbot/tools/api_requests.py:8]: I haven't added them either, but perhaps we should use type hints.

**Review by romanpeters — APPROVED** [review]

**Comment by romanpeters** [comment]
I'm still not sure how to use `get_json()` outside an async function. The example now returns a SyntaxError.

**Comment by romanpeters** [comment]
Is it possible to use `get_json()` outside an async function. The example returns a SyntaxError.

**Comment by RobinSikkens** [comment]
No it isn't, but as all command functions should be coroutines this shouldn't really be a problem.

**Pull request merged**